runtime/nls/cfre/cfrerr.nls

```diff
@@ -1482,7 +1482,7 @@ J9NLS_CFR_ERR_MULTIPLE_RECORD_ATTRIBUTES.user_response=Contact the provider of t
 # END NON-TRANSLATABLE
 
 # Record is not translatable
-J9NLS_CFR_RECORD_CLASS_CANNOT_BE_ABSTRACT=Record classes cannot be abstract
+J9NLS_CFR_RECORD_CLASS_CANNOT_BE_ABSTRACT=
```

Comment:
The explanation and system_action can be removed, keeping only the first (this) line.

Reply:
Removed against the suggestion above.

Reply:
These lines as well, since they contain anything between them.
# START NON-TRANSLATABLE
# END NON-TRANSLATABLE

Reply:
Removed.